docs/joss_paper/paper.md

```diff
@@ -54,63 +54,67 @@ bibliography: paper.bib
+If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical applications for tracking organ motion and other temporal changes require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expiratory and inspiratory phases, is such an example of both unsupervised (without labels) and combined supervision (trained with label dissimilarity based on segmentation of anatomical structures). Furthermore, registering prostate MR, acquired before surgery, and the intra-operative transrectal ultrasound images is an example of weakly-supervised training for multimodal image registration. Another DeepReg Demo illustrates MR-to-ultrasound image registration is to track tissue deformation and brain tumour resection during neurosurgery, in which combining unsupervised and weakly-supervised losses may be useful.
 
 ## Unpaired images
-Unpaired images are found in applications such as _single-modality_ _inter-subject_ registration, with two dataset loaders for labelled and unlabelled image data. Registering different MR images of brain from different subjects has played a fundamental role in population studies in neuroimaging. Two applications in registering unpaired inter-subject CT images, for lung and abdominal organs, represent the type of registration that may benefit from added label supervision. An example registering 3D ultrasound images from different surgical patients is included in DeepReg Demos, additionally, to demonstrate the support for experiments such as cross-validation in DeepReg.
+Unpaired images are found in applications such as _single-modality_ _inter-subject_ registration, with two dataset loaders for labelled and unlabelled image data. Registering different MR images of the brain from different subjects has played a fundamental role in population studies. Two applications which register unpaired inter-subject CT images, for lung and abdominal organs, represent the type of registration that may benefit from added label supervision. An example which registers 3D ultrasound images from different surgical patients is included in DeepReg Demos, additionally, to demonstrate the support for experiments such as cross-validation in DeepReg.
```

Comment:
I think the last sentence might be a bit easier to follow if we change from: 
"An example which registers 3D ultrasound images from different surgical patients is included in DeepReg Demos, additionally, to demonstrate the support for experiments such as cross-validation in DeepReg." 
To:
"Additionally, an example which demonstrates the support for experiments such as cross-validation in DeepReg has been included in DeepReg Demos. This examples registers 3D ultrasound images from different surgical patients."